src/Sylius/Bundle/CoreBundle/Command/Helper/CreateProgressBar.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\CoreBundle\Command\Helper;
+
+use Symfony\Component\Console\Helper\ProgressBar;
+use Symfony\Component\Console\Output\OutputInterface;
+
+trait CreateProgressBar
```

Comment:
I'm not a very big fan of traits, they always seem like hiding and implementation :/ Do we have any reason to not create a `ProgressBarCreator` service (or sth similar) and inject it into class where such a behaviour is needed?